tests/hubstorage/conftest.py

```diff
@@ -0,0 +1,175 @@
+import os
+import vcr
+import zlib
+import base64
+import pytest
+import pickle
+import requests
+from scrapinghub import HubstorageClient
+from scrapinghub.hubstorage.utils import urlpathjoin
+
+
+TEST_PROJECT_ID = "2222222"
+TEST_SPIDER_NAME = 'hs-test-spider'
+TEST_FRONTIER_NAME = 'test'
+TEST_FRONTIER_SLOT = 'site.com'
+TEST_BOTGROUP = 'python-hubstorage-test'
+TEST_COLLECTION_NAME = "test_collection_123"
+TEST_AUTH = os.getenv('HS_AUTH', 'f' * 32)
+TEST_ENDPOINT = os.getenv('HS_ENDPOINT', 'http://storage.vm.scrapinghub.com')
+
+# vcrpy creates the cassetes automatically under VCR_CASSETES_DIR
+VCR_CASSETES_DIR = 'tests/hubstorage/cassetes'
+
+
+class VCRGzipSerializer(object):
+    """Custom ZIP serializer for VCR.py."""
+
+    def serialize(self, cassette_dict):
+        # receives a dict, must return a string
+        # there can be binary data inside some of the requests,
+        # so it's impossible to use json for serialization to string
+        compressed = zlib.compress(pickle.dumps(cassette_dict))
+        return base64.b64encode(compressed).decode('utf8')
+
+    def deserialize(self, cassette_string):
+        # receives a string, must return a dict
+        decoded = base64.b64decode(cassette_string.encode('utf8'))
+        return pickle.loads(zlib.decompress(decoded))
+
+
+my_vcr = vcr.VCR(cassette_library_dir=VCR_CASSETES_DIR, record_mode='once')
+my_vcr.register_serializer('gz', VCRGzipSerializer())
+my_vcr.serializer = 'gz'
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--update-cassettes", action="store_true", default=False,
+        help="make tests with real services instead of vcr cassettes")
```

Comment:
What about adding another option that doesn't drop existing cassettes, just optionally ignores them? E.g. to check tests with real services, but not override existing cassettes without the need